pyproject.toml

```diff
@@ -28,8 +28,37 @@ imageio = "^2.33.1"
 pytest = "^7.2.1"
 pytest-cov = "^4.1.0"
 black = "^22.12.0"
+isort = "^5.11.4"
 teamcity-messages = "^1.32"
 mock = "^5.1.0"
+flake8 = "^6.0.0"
+pylint = "^2.15.10"
+
+[tool.black]
+line-length = 88
+target-version = ['py39']
+exclude = '''
```

Comment:
can you explain this for me face to face?

Reply:
This list contains the directories that should be skipped by black, for example you do not want to format external packages' code in your .venv directory, but only your own source code.

Reply:
thx!